components/pages/submission/Clinical/index.tsx

```diff
@@ -1,6 +1,6 @@
 /*
  *
- * Copyright (c) 2021 The Ontario Institute for Cancer Research. All rights reserved
+ * Copyright (c) 2024 The Ontario Institute for Cancer Research. All rights reserved
```

Comment:
After some recent licensing research, we learned these date updates aren't necessary.
```suggestion
 * Copyright (c) 2021 The Ontario Institute for Cancer Research. All rights reserved
```